### PR TITLE
Adding efs to terraform and startup template

### PR DESCRIPTION
**Pull request by smithatlanta**

### CODE REVIEW

File: tf_existing_net_rds_ec_ec2/main.tf [hunk 1]
```diff
@@ -12,8 +12,8 @@ terraform {
 # this is for an aws specific provider(not gcp or azure)
 provider "aws" {
   version = "~> 2.44.0"
-  region  = var.region
-  profile = var.profile
+  region  = "us-east-1"
+  profile = "aws-cloudarch-sandbox:aws-cloudarch-sandbox-admin"
```
Comment (ervazquez) [inline tf_existing_net_rds_ec_ec2/main.tf]: @smithatlanta Any reason why this is hard-coded into the terraform script?

File: tf_existing_net_rds_ec_ec2/securityGroups.tf [hunk 2]
```diff
@@ -135,26 +135,56 @@ resource "aws_security_group" "bastion_instance" {
...
   }
 
   tags = {
-    Name            = "${var.prefix}_bastion"
-    application     = var.tag_application
-    contact-email   = var.tag_contact_email
-    customer        = var.tag_customer
-    team            = var.tag_team
-    environment     = var.tag_environment
+    Name          = "${var.prefix}_efs"
```
Comment (ervazquez) [inline tf_existing_net_rds_ec_ec2/securityGroups.tf:183]: Do we still need the `_bastion` security group?

File: tf_existing_net_rds_ec_ec2/main.tf (hunk 1 above)
Comment (smithatlanta) [reply tf_existing_net_rds_ec_ec2/main.tf]: Thanks. Just fixed that.

File: tf_existing_net_rds_ec_ec2/securityGroups.tf (hunk 2 above)
Comment (smithatlanta) [reply tf_existing_net_rds_ec_ec2/securityGroups.tf:183]: Yes.  You will need this.

**Review by ervazquez — APPROVED** [review]

**Pull request merged**